src/Controls/src/Core/Platform/Windows/Extensions/BrushExtensions.cs

```diff
@@ -47,6 +47,7 @@ public static WBrush ToBrush(this Brush brush)
 			{
 				var wRadialGradientBrush = new WRadialGradientBrush()
 				{
+					GradientOrigin = new WPoint(radialGradientBrush.Center.X, radialGradientBrush.Center.Y),
```

Comment:
This changes are to fix a mistake creating the RadialGradientBrush.

Before
![radialbrush-before](https://user-images.githubusercontent.com/6755973/159532068-927abe38-2553-47f0-a9a7-947409d6ed17.PNG)
After
![radialbrush-after](https://user-images.githubusercontent.com/6755973/159532063-34dc5a30-72aa-4d6a-b4f4-88b62e81518c.PNG)